DEVELOPMENT.md

```diff
@@ -25,7 +28,15 @@ npx prettier --write "packages/**/src/**/*.ts"
 ```
 
 ## Run tests
-You can run all tests at the top level,
+First build a docker image of the project, 
```

Comment:
hmm, that's interesting, I've never actually needed to do this.  I do need to have docker installed and running, since the tests use pg-test, which in turn uses docker to spin up a postgres node.  But I haven't needed to manually run any docker images to run the tests locally, just having docker running was enough.

Reply:
You are right, just made the update